### ISSUE TITLE
Add json / logfmt format options  

### ISSUE THREAD

**Issue by RichardWright** [hg-pyun/axios-logger#80]
**Is your feature request related to a problem? Please describe.**
This is a handy lib, ideally it would plug into a standard display format as used by grafana (formats in here - https://grafana.com/docs/loki/latest/logql/#Metric-Queries)

**Describe the solution you'd like**
Return Json or logfmt

**Describe alternatives you've considered**
Writing long hand regex, but that seems pretty brittle.

**Comment by hg-pyun** [comment]
Good Idea. I'm using a Morgan now, so I'll check if it can be easily mashup.

**Comment by RichardWright** [comment]
I've written a change for this, but I need advice as it's definetly not ok as it stands.
